### ISSUE TITLE
Impress: Can't use keyboard keys (arrow, enter...) in the Insert Hyperlink popup

### ISSUE THREAD

**Issue by maks-w** [CollaboraOnline/online#7062]
### Describe the Bug

Can't use keyboard keys (arrow, enter...) in the Insert Hyperlink popup for a Presentation document. 
![image](https://github.com/CollaboraOnline/online/assets/1763638/8954da57-95e7-4e2a-b91e-f2e5f40a831f)

The focus switch to the slide panel
![image](https://github.com/CollaboraOnline/online/assets/1763638/29f70735-0896-43be-b04b-f7861c48a9b2)

### Steps to Reproduce

1. Open Impress
2. Click on Insert > Hyperlink
3. Write something in one of the inputs
4. Use arrow keys or delete key

### Expected Behavior

Moving the keyboard cursor in the input box

### Actual Behavior

Move slide focus if you use arrow keys or delete it if you use the delete key.

### Desktop

 - **OS:** Debian GNU/Linux 11 (bullseye)
 - **Browser:** Brave
 - **Version:** Version 1.56.20 Chromium: 115.0.5790.171 (Build officiel) (64 bits)
 
 ### Collabora version
 
- COOLWSD version: 23.05.2.2
- git hash: [912dfcd0](javascript:void(window.open('https://github.com/CollaboraOnline/online/commits/912dfcd0'));)
- LOKit version: Collabora Office 23.05.2.2
- git hash: [ec17398](javascript:void(window.open('https://hub.libreoffice.org/git-core/ec17398'));)
- Served by: Debian GNU/Linux 11 (bullseye)

**Comment by Ezinnem** [comment]
I can confirm this issue.

@eszkadev, please would you like to take a look?

**Comment by thebearon** [comment]
There was a similar issue with comments, which was a regression from this commit:
https://github.com/CollaboraOnline/online/commit/13387b8df6a35db78f4088bc0d6b945913791bc6
```
Author:     Gökay Şatır <gokaysatir@collabora.com>
AuthorDate: Mon Jan 23 12:36:14 2023 +0300
Commit:     Andras Timar <andras.timar@collabora.com>
CommitDate: Tue Jan 31 09:40:54 2023 +0100

    Enable deleting page / part with keyboard after a preview is clicked on.
```

This change was to enable cursor / delete / backspace key in the slides pane.

The fix in 22.05 was the following PR: #6306 (this particular issue didn't need to be fixed in 23.05, because the comment view changed)

Steps for this bug were:
- In a presentation with several slides, switch to eg. to the last,
- Reduce the window width, so you get the compact comment popup, and bring it up,
- Enter some text, and press backspace a few times.

-> Slides get deleted.

--------

Another issue from the same commit was:

- Open a presentation, (if it has a single slide, just add a few more)
- Switch to another slide using the slides pane,
- Click into a text box, and press the Del key.

-> Slide gets deleted.

This one got fixed via #6070.
